src/jni/platforms/jvm_desktop.cpp

```diff
@@ -38,12 +42,22 @@ namespace jni {
             args.options[i].optionString = (char*) initArgs.options[i].c_str();
         }
 
-        JavaVM* vm;
-        JNIEnv* env;
-        auto result = JvmLoader::get_create_jvm_function()(&vm, (void**) &env, (void*) &args);
+        JavaVM* java_vm{nullptr};
+        JNIEnv* jni_env{nullptr};
+
+#ifndef NO_USE_STDLIB
+        std::locale global;

```

Comment:
You just create a new global right? Can't you retrieve the one currently set ?

Reply:
From [cpp référence](https://en.cppreference.com/w/cpp/locale/locale/locale) default constructor of `std::local` will retreive global or use `std::local::classic` if global has not been set.